### PR TITLE
HWKMETRICS-368

### PR DESCRIPTION
**Pull request by jsanda**

### CODE REVIEW

File: core/metrics-model/src/main/java/org/hawkular/metrics/model/DataPoint.java [hunk 1]
```diff
@@ -59,15 +58,7 @@ public DataPoint(
         tags = Collections.emptyMap();
     }
 
-    @JsonCreator(mode = Mode.PROPERTIES)
-    public DataPoint(
```
Comment (stefannegrea) [inline core/metrics-model/src/main/java/org/hawkular/metrics/model/DataPoint.java]: I do not remember now but there was a reason for having just the constructor annotated. I really need to remember this ....

**Comment by stefannegrea** [comment]
This PR adds support for tagging individual data points, but does not add support for querying data points that are tagged with a specific tag. I am correct on the intent, then we just need to update the JIRA to reflect this.

**Comment by jsanda** [comment]
There is another ticket, [HWKMETRICS-373](https://issues.jboss.org/browse/HWKMETRICS-373), for querying and grouping by tag(s).

File: core/metrics-model/src/main/java/org/hawkular/metrics/model/DataPoint.java (hunk 1 above)
Comment (jsanda) [inline core/metrics-model/src/main/java/org/hawkular/metrics/model/DataPoint.java]: I spent the better part of the morning getting the JSON stuff working. Jackson throws an exception if more than one constructor is annotated with ```@JsonCreator```. I went back and tested some more with the field-level annotations removed. The integration tests pass so I am going to commit the changes.

File: core/metrics-model/src/main/java/org/hawkular/metrics/model/DataPoint.java [hunk 2]
```diff
@@ -35,13 +35,13 @@
  */
 public class DataPoint<T> {
 
-    @JsonProperty(value = "timestamp")
+//    @JsonProperty(value = "timestamp")
```
Comment (stefannegrea) [inline core/metrics-model/src/main/java/org/hawkular/metrics/model/DataPoint.java]: Can you please remove the commented lines if they are no longer needed?

**Comment by jsanda** [comment]
retest please

**Comment by jsanda** [comment]
retest please

**Comment by jsanda** [comment]
retest this please

**Comment by stefannegrea** [comment]
Thanks @jsanda!

**Pull request merged**